DESCRIPTION

```diff
@@ -8,18 +9,22 @@ Description: Provides easier interaction with
     returns an R data frame.
     Converts dates to 'POSIX' format.
     Manages throttling by 'Socrata'.
-Version: 1.6.1-2
-Date: 2015-6-5
-URL: https://github.com/Chicago/RSocrata
-BugReports: https://github.com/Chicago/RSocrata/issues
-Imports:
```

Comment:
It's a specious remove. The lines are "added" (i.e., kept) on lines 29 and 30 in the new file.